src/project/src/main/scala/io/giovannini/Main.scala

```diff
@@ -6,6 +6,12 @@ object Main extends App {
 
   val x = Test.test()
 
```

Comment:
🙅 [warning] method test in object Test is deprecated (since Lib 3.0): Do not use this method